packages/components/molecules/f-skeleton-loader/src/components/skeletons/CarouselCardRestaurantHeader.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div class="c-skeleton-loader-carouselCardRestaurantHeader c-skeleton-loader-bone">
+        <image-block />
+        <heading />
+        <text-block />
+        <text-block />
+    </div>
+</template>
+
+<script>
+import ImageBlock from './Image.vue';
+import TextBlock from './Text.vue';
+import Heading from './Heading.vue';
+
+export default {
+    components: { ImageBlock, Heading, TextBlock }
+};
+</script>
+
+<style lang="scss" scoped>
+
+$restaurant-logo-width: 48px;
+
+.c-skeleton-loader-carouselCardRestaurantHeader {
+    text-align: center;
+    background-color: #fff;
+    margin: auto;
+    width: 96%;
+    padding-bottom: spacing(x2);
+}
+
+.c-skeleton-loader-image {
+    margin: auto;
+    width: $restaurant-logo-width;
+    height: $restaurant-logo-width;
+    border: 1px solid white;
+    transform: translateY(-50%);
+}
+
+.c-skeleton-loader-heading {
+    margin: auto;
+    height: spacing(x3);
+    margin-bottom: spacing();
```

Comment:
I would use a shorthand here, or at least kept the rules for the same property as close together as possible.